uiowa/uiowa
# Feature: Adding event series link to event content type

Resolves https://github.com/uiowa/uiowa/issues/4010. 

## To Test

- `blt ds --site=default`
- `drush @default.local uli`
- Go to https://default.local.drupal.uiowa.edu/admin/config/development/configuration/config-split/event/enable
- `drush @default.local cim && drush @default.local cr`
- Add a "Series Link" and it should show up in the sidebar when the page is published.
- If there is content in the link field, the checkbox for direct linking should appear below the link field. 
  - If that box is checked, https://default.local.drupal.uiowa.edu/events should link directly to the Series Link url.
- Test on admissions events calendar using the link to series link field and verify that the item in the calendar view links directly to the url.
